### PR TITLE
Add instructions to install libblst manually

### PR DESCRIPTION
**Pull request by abailly-iohk**

### CODE REVIEW

File: docs/getting-started/install.md [hunk 1]
```diff
@@ -178,6 +178,34 @@ make check
 sudo make install
 ```
 
+#### Installing BLST
+
+Download and install BLST so that cardano-base can pick it up (assumes pkg-config is installed):
+
+```
```
Comment (carbolymer) [inline docs/getting-started/install.md:185]:
```suggestion
```bash
```
that would trigger better highlighting

**Review by carbolymer — APPROVED** [review]
Thanks for that!

**Review by olgahryniuk — APPROVED** [review]

File: docs/getting-started/install.md [hunk 2]
```diff
@@ -178,6 +178,34 @@ make check
...
+Description: Multilingual BLS12-381 signature library
+URL: https://github.com/supranational/blst
+Version: 0.3.10
+Cflags: -I${includedir}
+Libs: -L${libdir} -lblst
+EOF
+sudo cp libblst.pc /usr/local/lib/pkgconfig/
+sudo cp bindings/blst_aux.h bindings/blst.h bindings/blst.hpp  /usr/local/include/
+sudo cp libblst.a /usr/local/lib
+sudo chmod u=rw,go=r /usr/local/{lib/{libblst.a,pkgconfig/libblst.pc},include/{blst.{h,hpp},blst_aux.h}}
```
Comment (angerman) [inline docs/getting-started/install.md:206]: isn't this technically what the `install` command is supposed to do?

File: docs/getting-started/install.md [hunk 3]
```diff
@@ -178,6 +178,34 @@ make check
...
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: libblst
+Description: Multilingual BLS12-381 signature library
+URL: https://github.com/supranational/blst
+Version: 0.3.10
+Cflags: -I${includedir}
+Libs: -L${libdir} -lblst
+EOF
```
Comment (angerman) [inline docs/getting-started/install.md:202]: 🙏 I hope this doesn't bitrot over time 😬

**Review by angerman — APPROVED** [review]

File: docs/getting-started/install.md (hunk 2 above)
Comment (abailly-iohk) [reply docs/getting-started/install.md:206]: Yeah, except it doesn't 🤷

File: docs/getting-started/install.md (hunk 3 above)
Comment (abailly-iohk) [reply docs/getting-started/install.md:202]: I am sure it will, unfortunately. I have sent an upstream PR but not sure if or when it will be merged

**Review by CarlosLopezDeLara — APPROVED** [review]

**Pull request merged**